cgo14.go

```diff
@@ -24,7 +24,7 @@ func cgo(pkg *Package) ([]ObjTarget, []string) {
 	cgoCFLAGS = append(cgoCFLAGS, pcCFLAGS...)
 	cgoLDFLAGS = append(cgoLDFLAGS, pcLDFLAGS...)
 
-	if err := runcgo1(pkg, cgoCFLAGS, cgoLDFLAGS); err != nil {
+	if err := runcgo1(pkg, cgoCFLAGS, pcCFLAGS, cgoLDFLAGS); err != nil {
```

Comment:
cgo is not respecting the contents of cgoCFLAGS. Additionally, cgoCFLAGS is CFLAGS + "-g -02" + pcCFLAGS. -g -02 are not valid arguments for cgo which means we need to include pcCFLAGS separately when building the full cgo command.